### PR TITLE
Fix 30513: Fatal error because of access level

### PR DESCRIPTION
**Pull request by tfamula**
https://mantis.ilias.de/view.php?id=30513

### CODE REVIEW

**Comment by tfamula** [comment]
@alex40724: FYI

File: Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ class ilObjCertificateSettingsGUI extends ilObjectGUI
     /**
      * @var string
      */
-    private $type;
+    protected $type;
```
Comment (mjansenDatabay) [inline Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php]: @tfamula I guess we can remove this property completely. IMO it makes no sense that this is declared redundantly in the certificate class.

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]
Thx @tfamula , but we should delete the property instead.

File: Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php (hunk 1 above)
Comment (tfamula) [reply Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php]: @mjansenDatabay This was actually my first thought, but I was not completely sure about it :+1:

Comment (tfamula) [reply Services/Certificate/classes/class.ilObjCertificateSettingsGUI.php]: @mjansenDatabay Updated!

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**